yb-voyager/src/srcdb/postgres.go

```diff
@@ -1166,7 +1166,7 @@ const (
 		JOIN pg_namespace n ON e.extnamespace = n.oid
 		WHERE e.extname = 'pg_stat_statements'`
 
-	querySharedPreloadLibraries = `SELECT current_setting('shared_preload_libraries')`
+	queryCheckPgssLoaded = `SELECT query from pg_stat_statements LIMIT 1`
```

Comment:
schema of pgss table in case it's in non_public.

Reply:
good catch.